routes/product.js

```diff
@@ -48,11 +48,53 @@ const getProductByID = (request, response) => {
         });
 };
 
+const updateProduct = (request, response) => {
+    const {id,} = request.params;
+    const {
+        product_name, product_url, product_image, product_original_weight, product_original_weight_type, product_gram_weight, product_price, product_owner,
+    } = request.body;
+    ProductServices.updateProduct(
+        id, product_name, product_url, product_image, product_original_weight, product_original_weight_type, product_gram_weight, product_price, product_owner
+    )
+        .then(data => {
+            response.status(200).json({
+                'msg': `Successfully updated product.`,
+                data,
+            })
+        })
+        .catch(e => {
+            console.log(e);
+            response.status(400).json({
```

Comment:
@JayRodrig This is minor but I would recommend to look into it.

Can you double check if this is the appropriate status code to send back? I think it should be `500` not `400` since it catches errors that have occurred due to a failure within the server to update the product.

The same is true for the other error code 400's within this file and other files.